### ISSUE TITLE
Causative deinflection doesn't work

### ISSUE THREAD

**Issue by rbleuse** [mvysny/aedict#958]
Examples : 
suu -> suwaseru
taberu -> tabesaseru

<img src="https://user-images.githubusercontent.com/626863/118421082-4bd18180-b6fb-11eb-9e56-4a0c12907d9d.jpg" width="300" /><img src="https://user-images.githubusercontent.com/626863/118421085-4d9b4500-b6fb-11eb-8e25-e828acf1d7c4.jpg" width="300" />

**Comment by mvysny** [comment]
Good catch! The bug is in jputils which deinflects tabesaseru to tabesu. I'll create a bug report.

**Comment by mvysny** [comment]
Created a bug report: https://gitlab.com/mvysny/jputils/-/issues/3
